internal/observability/call.go

```diff
@@ -59,14 +59,25 @@ func (c call) EndWithAppError(err error, isApplicationError bool) {
 }
 
 func (c call) endLogs(elapsed time.Duration, err error, isApplicationError bool) {
-	msg := "Handled inbound request."
-	if c.direction != _directionInbound {
-		msg = "Made outbound call."
+	var ce *zapcore.CheckedEntry
+	if err == nil {
+		msg := "Handled inbound request."
+		if c.direction != _directionInbound {
+			msg = "Made outbound call."
+		}
+		ce = c.edge.logger.Check(zap.DebugLevel, msg)
+	} else {
+		msg := "Error handling inbound request."
+		if c.direction != _directionInbound {
+			msg = "Error making outbound call."
+		}
+		ce = c.edge.logger.Check(zap.ErrorLevel, msg)
 	}
```

Comment:
How about something like below?
```
direction := "inbound"
if c.direction != _directionInbound {
	direction = "outbound"
}
msg := "Handled " + direction + " request."
if err != nil {
	msg := "Error handling " + direction + " request."
}
ce = c.edge.logger.Check(zap.ErrorLevel, msg)
```

Reply:
Given that we want to create the `zapcore.CheckedEntry` based on the log-level, using an approach like this would require:
```
direction := "inbound"
if c.direction != _directionInbound {
	direction = "outbound"
}
var ce *zapcore.CheckedEntry
if err != nil {
	msg := "Error handling " + direction + " request."
	ce = c.edge.logger.Check(zap.ErrorLevel, msg)
} else {
	msg := "Handled " + direction + " request."
	ce = c.edge.logger.Check(zap.DebugLevel, msg)
}
```

Note that this would change the current behavior for successful requests. The previous `"Made outbound call"` would be replaced with `"Handled outbound call"`. This probably isn't a huge issue though, seeing as we are changing the behavior of our error messages.

Reply:
I think error messages like "handling inbound request" and "making outbound call" are more explanatory. Not sure which is more important - cleaner code or better error messages. @kriskowal do you have a preference here?

Reply:
Ah, nice - thanks for changing this! The code's nicer with `const` declarations at the top of the file. I was pointing out something a bit different, though - I didn't want us to add strings together in this function to put together the messages.